eng/Versions.props

```diff
@@ -38,10 +38,8 @@
   </PropertyGroup>
   <!--
     These versions should ONLY be updated by automation.
-
```

Comment:
Please undo all changes to this file and move `<AwsSdkS3Version>3.5.8.1</AwsSdkS3Version>` to `/eng/dependabot/Versions.props`.